project_billing_utils/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+##############################################################################
+#
+#    Author: Joël Grand-Guillaume
+#    Copyright 2010 Camptocamp SA
+#    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+##############################################################################
+
+{
+    'name': 'Analytic and project wizard for service companies',
+    'version': '11.0.1.0.0',
+    'category': 'Generic Modules/Projects & Services',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
+    'website': 'https://www.camptocamp.com',
```

Comment:
IMO Change website as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)